content/sensu-go/5.12/release-notes.md

```diff
@@ -36,6 +36,51 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.12.0 release notes
+
+**August 22, 2019** &mdash; The latest release of Sensu Go, version 5.12.0, is now available for download. There are some exciting feature additions in this release including the ability to output resources to a file from sensuctl and more granularity of control around checks/check hooks execution with an agent allow list. Additionally, this release includes the capability to delete assets and some more stability fixes around watcher functionality. See the [upgrading guide]() to upgrade Sensu to version 5.12.0.
+
+**IMPORTANT:**
+
+Due to changes in the release process, Sensu binary-only archives are now named following the pattern `sensu-go_5.12.0_$OS_$ARCH.tar.gz`, where $OS is the operating system name and $ARCH is the CPU architecture. These archives include all files in the top level directory. See the [installation guide][16] for the latest download links.
+Line item
+
+**NEW FEATURES:**
+
+Added sensu-agent and sensuctl binary builds for FreeBSD 
+Added sensuctl dump command to output resources to a file or STDOUT
+Added agent --allow-list to list allowed executables for checks and check hooks.
+
+**IMPROVEMENTS:**
+
+Licensed feature: Namespaces listed in both the web UI and sensuctl are now limited to those which the user has access to
+The event.check.name field has been added as a supported field selector
+Both the API and sensuctl can now be used to delete Assets
+The use of Protobuf serialization/deserialization over websocket can now be negotiated between agent and backend.
+We fixed some bugs in watchers through refactoring -- what does it mean to end users?
```

Comment:
Users who noticed checks failing to update or schedule should see an improvement.